requirements-btchip.txt

```diff
@@ -0,0 +1 @@
+git+https://github.com/LedgerHQ/btchip-python.git@fe292218e1c9c4939071650c729770bb6b34aa1b
```

Comment:
Can't use that tag as it's not up to date, I don't think there should be any disadvantage to using a particular commit. The library looks unmaintained anyway.

Regarding the ecdsa library it should be getting installed by setup.py:
`
    extras_require = {
	'smartcard': [ 'python-pyscard>=1.6.12-4build1', 'ecdsa>=0.9' ]
    },
`

From https://github.com/LedgerHQ/btchip-python/blob/master/setup.py although perhaps I need to pass something extra to pip to install 'extras'.